### DISCUSSION TITLE
Sort search results?

### DISCUSSION THREAD

**Discussion by TMac7373** [pymedusa/Medusa#10216] [Ideas & Feature requests]
when I'm adding a new show, please allow  me to sort the results by date.

**Comment by p0psicles** [comment]
Ok

**Reply by p0psicles** [reply]
I'm working on it

**Reply by p0psicles** [reply]
should be implemented in develop branch

**Reply by billyjeen22** [reply]
Hi, I'm using 1.0.16, has this feature made it to the main branch yet? I'd also really like this feature, but it doesn't seem to be working.

**Comment by OneCDOnly** [comment]
This is a great idea. When will it be included in the `master` branch please?
